x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-ebs-overview.json

```diff
@@ -0,0 +1,903 @@
+                "line_width": 1,
+                "metrics": [
+                  {
+                    "field": "aws.metrics.VolumeTotalReadTime.avg",
+                    "id": "61ca57f2-469d-11e7-af02-69e470af7417",
+                    "type": "avg"
+                  }
+                ],
+                "point_size": "5",
+                "separate_axis": 0,
+                "split_color_mode": "rainbow",
+                "split_mode": "terms",
+                "stacked": "none",
+                "terms_field": "aws.cloudwatch.dimensions.VolumeId",
+                "terms_order_by": "61ca57f2-469d-11e7-af02-69e470af7417"
+              }
+            ],
+            "show_grid": 1,
+            "show_legend": 1,
+            "time_field": "",
+            "type": "timeseries"
+          },
+          "title": "EBS Volume Total Read Time [Metricbeat AWS]",
+          "type": "metrics"
+        }
+      },
+      "id": "12eff7e0-b7b9-11e9-8349-f15f850c5cd0",
+      "migrationVersion": {
+        "visualization": "7.2.0"
+      },
+      "references": [],
+      "type": "visualization",
+      "updated_at": "2019-08-05T19:43:12.139Z",
+      "version": "WzY3MSwxMV0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "EBS Volume Idle Time [Metricbeat AWS]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "axis_formatter": "number",
+            "axis_position": "left",
+            "axis_scale": "normal",
+            "default_index_pattern": "metricbeat-*",
+            "default_timefield": "@timestamp",
+            "id": "61ca57f0-469d-11e7-af02-69e470af7417",
+            "index_pattern": "",
+            "interval": "5m",
+            "series": [
+              {
+                "axis_position": "right",
+                "chart_type": "line",
+                "color": "#68BC00",
+                "fill": "0",
+                "formatter": "s,s,3",
+                "id": "61ca57f1-469d-11e7-af02-69e470af7417",
+                "label": "Volume Idle Time",
+                "line_width": 1,
+                "metrics": [
+                  {
+                    "field": "aws.metrics.VolumeIdleTime.avg",
+                    "id": "61ca57f2-469d-11e7-af02-69e470af7417",
+                    "type": "avg"
+                  }
+                ],
+                "point_size": "5",
+                "separate_axis": 0,
+                "split_color_mode": "rainbow",
+                "split_mode": "terms",
+                "stacked": "none",
+                "terms_field": "aws.cloudwatch.dimensions.VolumeId",
+                "terms_order_by": "61ca57f2-469d-11e7-af02-69e470af7417"
+              }
```

Comment:
This volume idle time graph is an interesting one, why are values almost constant? would it make sense to set the axis min to zero?

Reply:
Good point on setting axis min to zero! Then the idle time line is almost straight... I'm not sure why though. Here is what I saw on Cloudwatch:
<img width="1333" alt="Screen Shot 2019-08-07 at 5 47 40 PM" src="https://user-images.githubusercontent.com/14081635/62665144-7e055b00-b93b-11e9-8167-608138266521.png">